### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by Phoebe-ovo** [tztztztztz/eqlv2#12]
Can you tell me how did you draw figure1: gradient ratio?
Is it the fitting curve?
If so, what fitting algorithm do you use?

**Comment by tztztztztz** [comment]
No, it is not a fitting curve.
I collect positive and negative gradients (of output logits) in the training for each category.

**Comment by Phoebe-ovo** [comment]
However, I notice that in Figure 2, it indeed is the fitting curve. And Figure 1 and 2 both show the accumulated gradients ratio of positives to negatives. at different training iteration. 
I want to know how do you apply the fitting?

**Comment by tztztztztz** [comment]
I don't understand the fitting curve you mentioned. Can you give me a further explanation?

**Comment by Phoebe-ovo** [comment]
![image](https://user-images.githubusercontent.com/47707811/129924733-6076fec8-1326-4c04-a380-ae1ba759caaa.png)
In this figure,  what are the meaning of the shallow and the deep curve?
Does the shallow means true value and deep means fitting value?
Thank you for your response

**Comment by tztztztztz** [comment]
Yes, I know what you mean. I use `gaussian_filter1d` to smooth those values. 

You can try with 
```

from scipy.ndimage.filters import gaussian_filter1d

gaussian_filter1d(data, sigma=5)

```

**Comment by Lucy0601** [comment]
Thank you for your response. I will try it! ---- Replied Message ----FromJingru ***@***.***>Date08/19/2021 14:31 ***@***.***> ***@***.******@***.***>SubjectRe: [tztztztztz/eqlv2] Question about the paper (#12)
Yes, I know what you mean. I use gaussian_filter1d to smooth those values.
You can try with

from scipy.ndimage.filters import gaussian_filter1d

gaussian_filter1d(data, sigma=5)



—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/tztztztztz/eqlv2/issues/12#issuecomment-901647086",
"url": "https://github.com/tztztztztz/eqlv2/issues/12#issuecomment-901647086",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Phoebe-ovo** [comment]
Thank you for your kind response, I will try it!
